code/game/gamemodes/game_mode.dm

```diff
@@ -191,7 +191,7 @@
 						msg += "<b>[L.name]</b> ([ckey(D.mind.key)]), the [L.job] ([span_boldannounce("Ghosted")])\n"
 						continue //Ghosted while alive
 
-
+	log_admin(msg.Join())
 	for (var/C in GLOB.admins)
 		to_chat(C, msg.Join())
```

Comment:
```suggestion
	var/concatenated_message = msg.Join()
	log_admin(concatenated_message)
	to_chat(GLOB.admins, concatenated_message)
```

Pretty wasteful to have to do a bunch of Join() operations (especially if there's like 10 admins on, that would have been 10 join operations), but this syntax works much better than a for-loop anyways. Caching the result means we only gotta do one join operation, which is much simpler than having to do two.